src/polygon/ZKEVMRollup.ts

```diff
@@ -95,8 +95,8 @@ export class ZKEVMRollup extends AbstractRollup<ZKEVMCommit> {
     proofSeq: ProofSequence
   ): HexString {
     return ABI_CODER.encode(
-      ['uint256', 'bytes[]', 'bytes'],
-      [commit.index, proofSeq.proofs, proofSeq.order]
+      ['tuple(uint256, bytes[], bytes)'],
```

Comment:
In multiple places. What was the reason for wholesale packaging these in tuples?

Reply:
* original ENS gateway had structs
* I liked using destructured tuple, but ran into "stack too deep" on more complex verifiers
* Newer verifiers used structs again (eg. PolygonPoS, OPReverse)
* Since this commit is breaking, I changed ALL verifiers to structs for consistency